src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,97 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+import wolox.training.exceptions.BookNonOwnedException;
+
+@Entity
+public class Users {
+    public Users() {
+        // default
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_SEQ")
+    @SequenceGenerator(name = "USER_SEQ", sequenceName = "USER_SEQ")
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    @NotNull
+    private String username;
+
+    @Column(nullable = false)
+    @NotNull
+    private String name;
+
+    @Column(nullable = false)
+    @NotNull
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_users",
```

Comment:
This annotation is only necessary if you want to rename column names. Otherwise, the annotation `@ManyToMany` should suffice.